packages/flutter/lib/src/cupertino/refresh.dart

```diff
@@ -268,6 +268,68 @@ typedef RefreshCallback = Future<void> Function();
 /// sliver such as [CupertinoSliverNavigationBar] and your main scrollable
 /// content's sliver.
 ///
+/// {@tool snippet}
```

Comment:
This will make the sample interactive. Feel free to check out more uses of {@tool dartpad... to see it in action. :)
```suggestion
/// {@tool dartpad --template=stateful_widget_cupertino_page_scaffold}
```